roles/apache/tasks/serverspec.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: serverspec checks for apache role
+  template: src={{ item }}
+            dest=/etc/serverspec/spec/localhost/
+            mode=0755
+  with_fileglob: ../templates/serverspec/*
```

Comment:
This role probably needs to depend on the server spec role, so that the directory tree gets built right.